cmd/nslookup.lisp

```diff
@@ -3,21 +3,21 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (import "lib/options/options.inc")
-(import "lib/net/utils.lisp")
-(import "lib/net/dns.lisp")
+(import "lib/net/utils.inc")
+(import "lib/net/dns.inc")
 
-(defun dns-type-to-string (type)
+(defun dns-type-to-string (dns_type)
 	; Convert DNS type number to string
 	(cond
-		((= type dns_type_a) "A")
-		((= type dns_type_ns) "NS")
-		((= type dns_type_cname) "CNAME")
-		((= type dns_type_soa) "SOA")
-		((= type dns_type_ptr) "PTR")
-		((= type dns_type_mx) "MX")
-		((= type dns_type_aaaa) "AAAA")
-		((= type dns_type_any) "ANY")
-		(t (str "TYPE" type))))
+		((= dns_type dns_type_a) "A")
+		((= dns_type dns_type_ns) "NS")
+		((= dns_type dns_type_cname) "CNAME")

```

Comment:
(= dns_type dns_type_cname) "CNAME"), a general comment, would identifiers like "CNAME" be better off using the self evaluation symbols ? like `:cname`